zipkin-storage/cassandra/src/main/resources/zipkin2-schema-indexes.cql

```diff
@@ -33,13 +33,14 @@ CREATE TABLE IF NOT EXISTS zipkin2.span_by_service (
     AND speculative_retry = '95percentile'
     AND comment = 'Secondary table for looking up span names by a service name.';
 
+DROP INDEX IF EXISTS zipkin2.span_annotation_query_idx;
```

Comment:
question.. I guess this means that `zipkin2.span (annotation_query)` results in an index named `zipkin2.span_annotation_query_idx`? so essentially we are re-creating that?

Reply:
second question: does dropping this cleanup the data associated? and third: if we create an index below, is there a command we can use to rebuild the index based on old data?

Reply:
first question: yes. re-creating it with different mode and analyzer_class
second question: yes it's drops the data. creating the new index will go over the base data and rebuild it into the new index.